src/components/html-text-input-ui/html-text-input-ui.android.js

```diff
@@ -39,37 +35,24 @@ export default class HTMLInputViewUI extends React.Component<PropsType, StateTyp
 		};
 	}
 
+	onContentSizeChange = ( event: NativeSyntheticEvent<TextInputContentSizeChangeEventData> ) => {
+		this.setState( { contentHeight: event.nativeEvent.contentSize.height } );
+	}
+
 	render() {
 		return (
-			<KeyboardAvoidingView style={ styles.container } parentHeight={ this.props.parentHeight }>
+			<KeyboardAvoidingView style={ styles.keyboardAvoidingView } parentHeight={ this.props.parentHeight }>
 				<ScrollView
 					style={ { flex: 1 } }
-					keyboardDismissMode="interactive" >
-					<TextInput
-						autoCorrect={ false }
-						textAlignVertical="center"
-						numberOfLines={ 1 }
-						style={ styles.htmlViewTitle }
-						value={ this.props.title }
-						placeholder={ this.props.titlePlaceholder }
-						onChangeText={ this.props.setTitleAction }
-					/>
-					<TextInput
-						autoCorrect={ false }
-						textAlignVertical="top"
-						multiline
-						style={ { ...styles.htmlView, height: this.state.contentHeight + 16 } }
-						value={ this.props.value }
-						onChangeText={ this.props.onChangeHTMLText }
-						onBlur={ this.props.onBlurHTMLText }
-						placeholder={ this.props.htmlPlaceholder }
-						scrollEnabled={ false }
-						onContentSizeChange={ ( event ) => {
-							this.setState( { contentHeight: event.nativeEvent.contentSize.height } );
-						} }
-					/>
+					keyboardDismissMode="interactive"
```

Comment:
We can also remove this keyboardDismissMode="interactive" because interactive is an ios only value, it must have been here because iOS used to have ScrollView too, it doesn't have affect on Android.